stake-pool/js/test/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { Connection, PublicKey } from "@solana/web3.js";
+import BN from "bn.js";
+import { StakePoolAccount, getStakePoolAccounts } from "../src";
+
+export function isStakePoolAccount(account: any): account is StakePoolAccount {
+  return (account !== undefined) &&
+    (account.account !== undefined) &&
+    (account.account.data !== undefined) &&
+    ('manager' in account.account.data);
+}
+
+export async function getFirstStakePoolAccount(
+  connection: Connection,
+  stakePoolProgramAddress: PublicKey,
+): Promise<StakePoolAccount | undefined> {
+  const accounts = await getStakePoolAccounts(connection, stakePoolProgramAddress);
+
+  return accounts!
+    .filter(account => isStakePoolAccount(account))
+    .pop() as StakePoolAccount;
+}
```

Comment:
nit: since these won't be needed after the integration tests are taken out, this file could be renamed to `equal.ts`